src/cascadia/TerminalApp/App.cpp

```diff
@@ -661,6 +661,7 @@ namespace winrt::TerminalApp::implementation
     // - Close the currently focused tab. Focus will move to the left, if possible.
     void App::_CloseFocusedTab()
     {
+        // TODO: MSFT:21268795: Need a better story for what should happen when the last tab is closed.

```

Comment:
This shouldn't be an internal MSFT work-item todo anymore. Port the internal issue to GitHub or please reference a GitHub issue ID instead. (and below)